src/test/unit/spec1.js

```diff
@@ -0,0 +1,31 @@
+
+
+
+
+describe("app", function () {
+
+  let $rootScope;
+  let $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["beforeEach" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879981/issues/source?bid=3233857&fileBranchId=3284028#l11)